src/model.jl

```diff
@@ -481,24 +506,6 @@ function get_consumer_benchmark(c::ConsumerRef)
     end
 end
 
-# function get_consumer_total_endowment(jm, c::ConsumerRef)
-#     m = c.model
-
-#     endowments = []
-#     for d in m._demands
-#         if d.consumer == c
-#             push!(endowments, :(
-#                 +($((:($(en.quantity) * 
-#                 $(en.commodity)) for en in d.endowments)...))
-#             ))
-#         end
-#     end
-#     total_end = :(+(0., $(get_tax_revenue_for_consumer(jm, m, c)),  $(endowments...)))
-
-#     return total_end
-
-# end
-
```

Comment:
Another delete of redundant lines, this time it had already been commented out (bc I was afraid it would be necessary after all).